tests/ui/accessibility.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { test, expect } from 'playwright/test';
+import AxeBuilder from '@axe-core/playwright'; 
+const callAPI = require("./api-call.spec.ts");
+
+
+test.describe('homepage', () => { 
+  test('should not have any automatically detectable accessibility issues', async ({ page }) => {
+    await page.goto('/'); 
+
+    const accessibilityScanResults = await new AxeBuilder({ page }).analyze(); 
```

Comment:
Extract accessibility testing to utility function to not repeat in every possible test case